src/main.rs

```diff
@@ -118,6 +147,7 @@ fn main() -> crate::common::CliResult {
     if !matches!(
         cli_cmd,
         Ok(Some(CliCmd {
+            offline: _,
```

Comment:
We don't need to match all the specific fields, so we can just use `..` here (probably move after `top_level` field):

```suggestion
            ..
```